charts/actions-runner-controller/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.23.3
+version: 0.24.0
```

```suggestion
version: 0.23.3
```

It would be great if you could let us batch some more changes before the next minor release!